packages/SwingSet/src/controller/initializeSwingset.js

```diff
@@ -67,6 +69,22 @@ export async function buildKernelBundles(options = {}) {
   return harden(bundles);
 }
 
+// Unit tests can call this to amortize the bundling costs: pass the
+// result to initializeSwingset's initializationOptions.kernelBundles
+// (for the vat/device/worker bundles), and you can pass .kernelBundle
+// individually to makeSwingsetController's
+// runtimeOptions.kernelBundle
+
+// Tests can also pass the whole result to buildVatController's
+// runtimeOptions.kernelBundles, which will pass it through to both.
+
+export async function buildKernelBundles() {
+  const bp = buildVatAndDeviceBundles();
+  const kp = buildKernelBundle();
+  const [vdBundles, kernelBundle] = await Promise.all([bp, kp]);
+  return harden({ kernelBundle, ...vdBundles });
```

Comment:
When the value is passed into `runtimeOptions`, I wanted it to be named `kernelBundle` and not `kernel` (since at that point it's no longer living inside a container with `bundles` in the name). I want to keep using `buildKernelBundles` and `buildVatController(.. initializationOptions.kernelBundles)` names to avoid rewriting a bunch of other tests.

I guess one option would be to have `buildKernelBundles` and `buildVatController` create/accept `.kernel`, and then rename it within `buildVatController` to pass `runtimeOptions.kernelBundle` into `makeSwingsetController`. Lemme see how that looks.